python/ray/data/dataset.py

```diff
@@ -2830,21 +2832,43 @@ def to_tf(
     ) -> "tf.data.Dataset":
         """Return a TF Dataset over this dataset.
 
-        The TF Dataset will be created from the generator returned by the
-        ``iter_batches`` method. ``prefetch_blocks`` and ``batch_size``
-        arguments will be passed to that method.
-
-        This is only supported for datasets convertible to Arrow records.
```

Comment:
Is this still true?

Reply:
I don't think so, since we now have the NumPy narrow waist. I think this can be removed.